src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -71,40 +72,43 @@ public void CanGetAllTimesAfterCreation()
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Linux)] // Windows tested below, and OSX does not currently support millisec granularity
-        public void TimesIncludeMillisecondPart_Linux()
+        [PlatformSpecific(TestPlatforms.AnyUnix)] // Windows tested below, and OSX HFS driver format does not support millisec granularity
+        public void TimesIncludeMillisecondPart_Unix()
         {
             T item = GetExistingItem();
 
             string driveFormat = new DriveInfo(GetItemPath(item)).DriveFormat;
-
-            Assert.All(TimeFunctions(), (function) =>
+            if (!PlatformDetection.IsOSX || 
```

Comment:
Style nit, sometimes `!(A && B)` is easier to read than `!A || !B`. In this case I read "if it's not Mac or it's not HFS" whereas what a human would say is "if it's not HFS on Mac" so I believe `!(PlatformDetection.IsOSX && driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase)` may be easier to read.

Reply:
No need to go back to fix, just general observation